question/bank/exportquestions/classes/exportquestions_helper.php

```diff
@@ -0,0 +1,57 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Library functions used by qbank_exportquestions/export.php.
+ *
+ * @package    qbank_exportquestions
+ * @copyright 2021, Guillermo Gomez Arias <guillermogomez@catalyst-au.net>
```

Comment:
one more space please/
also can you please add the info of which code its based on.